dump/src/main.rs

```diff
@@ -25,8 +25,8 @@ fn os_compatibility() -> Result<(), ()> {
 /// Dump the contents of DICOM files
 #[derive(Debug, StructOpt)]
 struct App {
-    /// The DICOM file to read
-    file: PathBuf,
+    /// The DICOM file(s) to read
+    files: Vec<PathBuf>,
```

Comment:
Let's make this argument required, so that an error is raised if the user tries to run this without passing any files.
```suggestion
    /// The DICOM file(s) to read
    #[structopt(required = true)]
    files: Vec<PathBuf>,
```

Reply:
Sure, I've addressed this.